core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGeneData.java

```diff
@@ -504,6 +514,12 @@ else if (i.contains("gene_name")) {
     public static boolean updateLength(String symbol, String chromosome, List<long[]> loci, int refreneceGenomeId) throws IOException, DaoException {
         DaoGeneOptimized daoGeneOptimized = DaoGeneOptimized.getInstance();
         CanonicalGene gene = daoGeneOptimized.getNonAmbiguousGene(symbol);
+        
+        if (gene==null) {
+            ProgressMonitor.logWarning("Unable to retrieve gene by symbol: " +symbol);
+            return false;
+        }
+
         System.out.println(" --> update reference genome gene:  " + gene.getHugoGeneSymbolAllCaps());
         DaoReferenceGenomeGene daoReferenceGenomeGene = DaoReferenceGenomeGene.getInstance();
         boolean lengthUpdated = false;
```

Comment:
move line 525 before line 518, then line 520 just be
```
return lengthUpdated
```

Reply:
Although I fail to see the added value of it, I did implement this.
Done.